README.md

```diff
@@ -17,7 +17,7 @@ npm install dialog-size
     ```scss
     @import 'dialog-size/dialog-size';
     ```
-    PS: make sure to **add `node_modules` to your [import paths](https://github.com/sass/node-sass#includepaths)**
```

Comment:
No reason to have this bold. It is very unreadable where there is code and a hyperlink.